### PR TITLE
Pi-hole core v5.0

### PR DESCRIPTION
**Pull request by DL6ER**
**Release**

Further information may be found in the blog post and release notes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -175,7 +175,7 @@ While quite outdated at this point, [this original blog post about Pi-hole](http
 -----
 
 ## Coverage
-- [Lifehacker: Turn A Raspberry Pi Into An Ad Blocker With A Single Command](https://www.lifehacker.com.au/2015/02/turn-a-raspberry-pi-into-an-ad-blocker-with-a-single-command/) (Feburary, 2015)
+- [Lifehacker: Turn A Raspberry Pi Into An Ad Blocker With A Single Command](https://www.lifehacker.com.au/2015/02/turn-a-raspberry-pi-into-an-ad-blocker-with-a-single-command/) (February, 2015)
```
Comment (XhmikosR) [inline README.md:178]: @DL6ER maybe these should be updated from the current docs page? https://github.com/pi-hole/docs/blob/master/docs/main/coverage.md

**Comment by XhmikosR** [comment]
BTW it seems there are 2 strickler-ci errors, just pointing out in case it was missed :)

File: README.md (hunk 1 above)
Comment (dschaper) [reply README.md:178]: Just take the whole Coverage section out. Link it to docs. The README is bloated anyways.

**Comment by DL6ER** [comment]
The sticker errors can be ignored for now. We control the content of this variable. We can address this with the next release.

**Pull request merged**